pkg/shell/hosts_dialog.jsx

```diff
@@ -479,18 +484,27 @@ class HostKey extends React.Component {
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
                 <p>{_("The resulting fingerprint is fine to share via public methods, including email.")}</p>
-                <p>{_("If the fingerprint matches, click 'Accept key and connect'. Otherwise, do not connect and contact your administrator.")}</p>
+                <p>{_("If the fingerprint matches, click 'Trust and add host'. Otherwise, do not connect and contact your administrator.")}</p>
             </>;
         } else {
+            const fingerprint_help = <Popover bodyContent={
+                <>
+                    <p>{ _("The resulting fingerprint is fine to share via public methods, including email.") }</p>
+                    <p>{ _("If you are asking someone else to do the verification for you, they can send the results using any method.") }</p>
+                </> }>
+                <OutlinedQuestionCircleIcon />
+            </Popover>;
             body = <>
                 <p>{cockpit.format(_("You are connecting to $0 for the first time."), this.props.host)}</p>
-                <p>{_("To ensure that your connection is not intercepted by a malicious third-party, please verify the host key fingerprint:")}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
-                <p className="hostkey-type">({key_type})</p>
-                <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
-                <p>{_("The resulting fingerprint is fine to share via public methods, including email.")}</p>
-                <p>{_("If the fingerprint matches, click 'Accept key and connect'. Otherwise, do not connect and contact your administrator.")}</p>
+                <ExpandableSection toggleText={ _("Verify fingerprint") }
+                                   isExpanded={this.state.verifyExpanded}
+                                   onToggle={(_ev, verifyExpanded) => this.setState({ verifyExpanded }) }>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19409-20231010-153219-d8bb31b4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Added a test.